docs/manuals/core/config/scenario-config.md

```diff
@@ -15,77 +17,116 @@ A scenario configuration is necessary to instantiate a [Scenario](../concepts/sc
 
     Note that we cannot use the word "_entity_owner" as a key in the properties as it has been reserved for internal use.
 
-# From pipeline configs
+# From task configs
+
+Here is a simple example assuming the task configuration `multiply_task_cfg` has already been created:
+
+```python linenums="1"
+from taipy import Config
+
+scenario_cfg = Config.configure_scenario("multiply_scenario",
+                                         [multiply_task_cfg])
```

Comment:
Isn't there a syntax where, if there's only one task, you don't need the array construct?
(please think of the developer and save him time)

Reply:
@jrobinAV 
We dont have that yet, but that's certainly true! Should we create a ticket for this so we won't forget about it?

Reply:
yes let s create a ticket